src/store/index.js

```diff
@@ -8,8 +8,9 @@ Vue.use(Vuex);
 export default new Vuex.Store({
   state: {
     movies: [],
-    searchQuery: '',
+    searchQuery: "",
```

Comment:
Don't keep the search query input state in the store.

Instead, pass it as a parameter to the VueX action.

Reply:
Since we need this property for scrolling, I removed it from the data, but left it in the state.